docs/changes.rst

```diff
@@ -135,66 +135,66 @@ Compatibility notes:
 
 - init: the --encryption argument is mandatory now (and there are several choices).
 - moved "borg migrate-to-repokey" to "borg key migrate-to-repokey".
-- "borg change-passphrase" is deprecated, use "borg key change-passphrase" instead.
+  "borg change-passphrase" is deprecated, use "borg key change-passphrase" instead.
```

Comment:
I saw it as one change, introducing the new name and deprecating the old one in the same move